app/pages/OrganizationListingPage.jsx

```diff
@@ -80,6 +81,11 @@ export class OrganizationListingPage extends React.Component {
     // console.log(resource, window.google);
     return (!resource || !window.google ? <Loader />
       : (
+        <div>
```

Comment:
Rather than creating a `<div>` to wrap everything just for the sake of being able to put the `<Helmet>` component somewhere, let's use [`<React.Fragment>`](https://reactjs.org/docs/fragments.html), which is the official way of allowing a component to return multiple sibling components without needing to create an unnecessary HTML `<div>` element.